mxcubecore/HardwareObjects/mockup/XRFSpectrumMockup.py

```diff
@@ -17,53 +17,28 @@
 #  You should have received a copy of the GNU Lesser General Public License
 #  along with MXCuBE. If not, see <http://www.gnu.org/licenses/>.
 
-import time
+"""Mockup file to define _execute_xrf_spectrum method
+"""
+
+from time import sleep
 
 from mxcubecore.HardwareObjects.abstract.AbstractXRFSpectrum import (
     AbstractXRFSpectrum,
 )
-from mxcubecore.BaseHardwareObjects import HardwareObject
 
 
-spectrum_test_data = [
-    0,
-    20,
-    340,
-    70,
-    100,
-    110,
-    120,
-    200,
-    200,
-    210,
-    1600,
-    210,
-    200,
-    200,
-    200,
-    250,
-    300,
-    200,
-    100,
-    0,
-    0,
-    0,
-    90,
-]
+__copyright__ = """ Copyright © 2010-2023 by the MXCuBE collaboration """
```

Comment:
Skip the years maybe?

```suggestion
__copyright__ = """ Copyright © by the MXCuBE collaboration """
```

Reply:
yes, you are right. I've missed this one